extensions/windows/objects/startup_app.json

```
@@ -0,0 +1,12 @@
+{
+  "caption": "Startup Application",
+  "name": "startup_app",
+  "description": "The startup application object describes an application that has associated startup criteria and configurations.",
+  "extends": "startup_app",
+  "attributes": {
+    "win_service": {
+      "description": "The startup application windows service resource.",
+      "requirement": "optional"
+    }
+  }
```

```suggestion
  },
  "constraints": {
    "just_one": [
      "driver",
      "job",
      "process",
      "win_service"
    ]
  }
```
This will override the `constraints` in `startup_app` and replace them with this one that has `win_service` added.